pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -150,24 +162,24 @@ func NewQueue(kubeClient client.Client, recorder events.Recorder, cluster *state
 	queue := &Queue{
 		// nolint:staticcheck
 		// We need to implement a deprecated interface since Command currently doesn't implement "comparable"
-		TypedRateLimitingInterface: workqueue.NewTypedRateLimitingQueueWithConfig(
-			workqueue.NewTypedItemExponentialFailureRateLimiter[*Command](queueBaseDelay, queueMaxDelay),
-			workqueue.TypedRateLimitingQueueConfig[*Command]{
-				Name: "disruption.workqueue",
-			}),
-		providerIDToCommand: map[string]*Command{},
-		kubeClient:          kubeClient,
-		recorder:            recorder,
-		cluster:             cluster,
-		clock:               clock,
-		provisioner:         provisioner,
+		source:               make(chan event.TypedGenericEvent[*v1.NodeClaim], 10000),
+		providerIDToCommand:  map[string]*Command{},
+		nodeClaimsInQueueSet: make(sets.Set[string]),
```

Comment:
nit: style, more standard syntax
```suggestion
		nodeClaimsInQueueSet: sets.New[string](),
```